modules/ossm-security-mtls.adoc

```diff
@@ -99,3 +120,19 @@ The default is `TLS_AUTO` and does not specify a version of TLS.
 |`TLSv1_3`
 |TLS version 1.3
 |===
+
+[id="ossm-security-enabling-controlplane_{context}"]
+== Enabling strict mTLS for Mixer telemetry or policy components
+
+Secure connections are always used when proxies communicate with the control plane regardless of the `spec.security.controlPlane.mtls` setting. If you are using Mixer telemetry or policy set `spec.security.controlPlane.mtls` to `true` in your `ServiceMeshControlPlane` resource to enable strict mTLS.
```

Comment:
I'm still struggling with the last sentence just a bit - probably because I don't know what Mixer telemetry/policy means. But maybe it just needs a comma after "If you are using Mixer telemetry or policy, set" ?

Also here and in the note above, consider changing the instances of "are using" to just "use".